admin_guide/manage_rbac.adoc

```diff
@@ -631,6 +631,9 @@ cluster role bindings use non-namespaced resources.
 |`$ oc adm policy remove-cluster-role-from-group _<role>_ _<groupname>_`
 |Removes a given role from specified groups for all projects in the cluster.
 
+|`--rolebinding-name=`
+|Can be used with `oc adm policy` commands to retain the rolebinding name assigned to a user.
+
```

Comment:
The option `--rolebinding-name=` could be apply for not only user, but also for group.
And it should be added to table 1 also.

Reply:
Thanks @stuartchuan , I added `assigned to a user or group` and added it to table 1 as well.